compiler/src/dotty/tools/dotc/transform/Pickler.scala

```diff
@@ -86,7 +86,7 @@ class Pickler extends Phase {
         pickled
       }(using ExecutionContext.global)
       def force(): Array[Byte] = Await.result(pickledF, Duration.Inf)
-      if ctx.settings.YtestPickler.value then force()
+      if ctx.settings.YtestPickler.value || ctx.mode.is(Mode.Interactive) then force()
```

Comment:
why does interactive mode need to be  special-cased ?

Reply:
The Repltests failed with an error message that seems to have nothing to do with the change (illegal cast from Int to Unit). I don't know the ReplTests well enough to find out what goes on, and don't have time to find out.